install.rdf

```diff
@@ -1,40 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-     xmlns:em="http://www.mozilla.org/2004/em-rdf#">
-
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
   <Description about="urn:mozilla:install-manifest">
-    <em:id>exchangecalendar@extensions.1st-setup.nl</em:id>
-    <em:version>4.0.0-beta4</em:version>
+    <em:id>exchangecalendar@ExchangeCalendar</em:id>
+ <em:version>4.0.0-beta5</em:version>
```

Comment:
note: it would be neat if this could be a template that could be filled in at build time so the version data, etc could be easily kept in sync.